src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -203,15 +203,15 @@ def test_wraps_the_corresponding_method_of_the_held_interface(
 
 
 class Test_Manager_flushCaches:
-    def test_method_defined_in_python(self, method_introspector):
-        assert method_introspector.is_defined_in_python(Manager.flushCaches)
+    def test_method_defined_in_cpp(self, method_introspector):
+        assert not method_introspector.is_defined_in_python(Manager.flushCaches)
         assert method_introspector.is_implemented_once(Manager, "flushCaches")
 
     def test_wraps_the_corresponding_method_of_the_held_interface(
         self, manager, mock_manager_interface, a_host_session
     ):
         method = mock_manager_interface.mock.flushCaches
-        assert manager.flushCaches() == method.return_value
+        assert manager.flushCaches() is None
```

Comment:
[Consider] Is it necessary to test for `None` at all now? Not sure we do this for other void returns.

Reply:
It was just a way that I could still use the word `assert` in the test case, rather than just calling the function. You're right though, we otherwise don't test void returns.  I'll get rid of it.

Reply:
Updated as part of d0c780cb